test/test_networks/test_conditional_hint_network.jl

```diff
@@ -14,11 +14,16 @@ batchsize = 2
 L = 2
 K = 2
 
-# Multi-scale and single scale network
-CH0 = NetworkMultiScaleConditionalHINT(n_in, n_hidden, L, K; split_scales=false, k1=3, k2=1, p1=1, p2=0)
-CH1 = NetworkConditionalHINT(n_in, n_hidden, L*K; k1=3, k2=1, p1=1, p2=0)
+# single scale network
+CH0 = NetworkConditionalHINT(n_in, n_hidden, L*K; k1=3, k2=1, p1=1, p2=0)
```

Comment:
L*K ?

Reply:
I believe that is how you can make the two networks sort of equivalent in depth. Each scale step L has K flow steps so a total of L*K depth.

Reply:
Olld the current test shouldn't be rmeoved/modified. If you add a new functionality you add an extra test,

Reply:
As far as I know, I am not modifying any previous tests, I am only adding a test for a reversed multiscale conditional and for logdets turned off on multiscale and nonmultiscale conditional.

Reply:
Will make a new file so this is resolved.